Sources/Sentry/SentryIOTrackingIntegration.m

```diff
@@ -0,0 +1,26 @@
+#import "SentryIOTrackingIntegration.h"
+#import "SentryNSDataSwizzling.h"
+#import "SentryOptions.h"
+
+@interface
+SentryIOTrackingIntegration ()
+
+@property (nonatomic, strong) SentryOptions *options;
+@end
```

Comment:
`l`: We don't do anything with options in this class, so I think we can remove the property.
```suggestion
```

Reply:
Should we add an automatic trace IO option?

Reply:
Yes, maybe we can even prefix it with `experimental` so we can ship it disabled with an official release to avoid having a beta and alpha. Do you know if they already have a name for that on Android? `enableFileTracking`?

Reply:
I didn't find any IO related option for android.

@romtsn can confirm it.

Reply:
`l`: We never make use of the property in this class. I think we can remove it.